static/js/components/SourceList.jsx

```diff
@@ -43,17 +49,40 @@ const useStyles = makeStyles((theme) => ({
   },
   blockWrapper: {
     width: "100%",
+    marginBottom: "0.5rem",
   },
   title: {
-    margin: "1rem 0rem 0rem 0rem",
+    margin: "0.5rem 0rem 0rem 0rem",
   },
   spinner: {
     marginTop: "1rem",
   },
 }));
 
+const getStyles = (classification, selectedClassifications, theme) => {
+  return {
+    fontWeight:
+      selectedClassifications.indexOf(classification) === -1
+        ? theme.typography.fontWeightRegular
+        : theme.typography.fontWeightMedium,
+  };
+};
+
+const filterOutEmptyValues = (data) => {
+  let filteredData = {};
+  // Filter out empty fields from an object (form data)
+  Object.keys(data).forEach((key) => {
+    // Empty array ([]) counts as true, so specifically test for it
+    if (!(Array.isArray(data[key]) && data[key].length == 0) && key) {
+      filteredData[key] = data[key];
+    }
+  });
+  return filteredData;
+};
+
```

Comment:
There were issues associated with passing along empty/default form values introduced by #1487 (the ducks used to just ignore form values that weren't filled out but now they're added as query params I think), and this is used to remove those before calling the API fetch action

Reply:
@kmshin1397  Would it suffice to just add a line to `API.GET` that removes any properties whose value are in `[null, undefined, ""]` or something, before passing it to URLSearchParams?

Reply:
i.e. just always pass `filterParams` through the above function in API.GET?

Reply:
@acrellin I think that's a good idea - I can imagine this popping up again as an issue elsewhere in the future if we're not careful. And I don't think there will ever be a case where we need to let those through. I'll go ahead and make that change.